core/primitives/src/types.rs

```diff
@@ -65,6 +65,37 @@ pub enum BlockId {
     Hash(CryptoHash),
 }
 
+// 1. Transaction structs.
```

Comment:
hmm, is this supposed to be in this PR? Seemed to work fine for me without DevNet changes.

Reply:
yep, it's left from merge. deleted.